src/main/resources/native/android/c/launcher.c

```diff
@@ -53,12 +64,30 @@ char** createArgs() {
     strcat(tmpArgs,appDataDir);
     result[origSize]=tmpArgs;
     argsize++;
+LOGE(stderr, "CREATE ARGS done");
     return result;
 }
+
+void registerMethodHandles (JNIEnv *aenv) {
+    activityClass = (*aenv)->NewGlobalRef(aenv, 
+          (*aenv)->FindClass(aenv, "com/gluonhq/helloandroid/MainActivity"));
```

Comment:
do we need this class in the jni config file? 
Also, do we have to replace this package name before compiling?

Reply:
no, this method is called from the JNI_OnLoad from the Android code, hence it is running in the Android VM (not in the GraalVM).
The package name here is unrelated to the package name of the developer application, so this can be constant.